sub/sd_ass.c

```diff
@@ -1094,11 +1113,10 @@ bstr sd_ass_pkt_text(struct sd_filter *ft, struct demux_packet *pkt, int offset)
     return txt;
 }
 
-bstr sd_ass_to_plaintext(char *out, size_t out_siz, const char *in)
+bstr sd_ass_to_plaintext(char **out, const char *in)
 {
-    struct buf b = {out, out_siz, 0};
+    bstr b = {*out};
```

Comment:
```suggestion
    bstr b = bstr0(*out);
```
bstr has to have size.

Reply:
This is deliberate; we want to start with a 0-size (but preallocated) `bstr`; `bstr0` would call `strlen` on the input and we'd end up appending to the end instead of overwriting.

Reply:
Ok. One more question, why this function returns bstr that is never used and instead char** is overwritten?

Reply:
It's just because the existing implementation already did. I can remove it if you feel strongly.